dls-controls/malcolmjs
# Bugfix/fix screenshots

## Description

Updated the screenshot scripts

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Review the new images for the user guide

## Agile board tracking
